### ISSUE TITLE
[Core][♿]: add keyboard navigation / interaction

### ISSUE THREAD

**Issue by mimshins** [sonnat/sonnat-ui#3]
We have to make sure that controls can be operated using the keyboard. Native interactive HTML elements, such as links and form controls, support keyboard accessibility by default, and these should be used wherever possible. However, many custom controls and widgets use elements such as div, span, and li, which do not by default provide keyboard support. In this case, keyboard support must be supplied by Sonnat.

List of things we need to make sure of:
- Make sure every active control is in focus order
- Make sure currently inactive controls are not in focus order
- Make sure that active controls are operable using the keyboard
- Make sure that content that appears/disappears, is designed such that users can perceive the additional content and dismiss it without disrupting the page experience

Basic keyboard controls on the web:
- `Tab` navigates to next focusable element
- `Shift+Tab` navigates to previous focusable element
- `Arrows` navigate between related radio buttons, menu items, or widget items
- `Enter` activates a link or button, or submits a form
- `Space` activates a button or toggle
- `Esc` closes menus, modals, and other popover variations

**Comment by mimshins** [comment]
[🚀] Merged!
[🔗] [Link to the Log](https://github.com/sonnat/sonnat-ui/releases/tag/v1.3.0)
